SabPicariello/io-app-test
# Test per nuovo profilo e e2e per archiviazione

## Short description

- Added test for new profile page
- Added e2e test for archiving and unarchiving process


## How to test
You can simply run the tests to verify if they pass:

- `detox test --configuration ios.sim.release ts/__e2e__/archiveMessages.e2e.ts`
- `jest -i "ts/screens/profile/__test__/NuovoProfilo.test.tsx"`
